x-pack/test/detection_engine_api_integration/utils.ts

```diff
@@ -615,7 +617,9 @@ export const createSignalsIndex = async (
   await countDownTest(
     async () => {
       await supertest.post(DETECTION_ENGINE_INDEX_URL).set('kbn-xsrf', 'true').send();
-      return true;
+      return {
```

Comment:
I can't leave a comment on the line number but since we have updated the default count from 20 to 50 here: https://github.com/elastic/kibana/pull/130052/files#diff-a180e28a6c1d993fab6c2be0586f31558807b9e2feb0dba56886f43c5d97fcceR1017

we should probably update the comment on this function as well to reflect that change.

Reply:
Oh thanks, I will update the comment and then merge. Appreciate it @dhurley14 !

Reply:
There were a few more spots and one other area that had 20, so I wanted to make them all an even 50 as 20 seemed a little low to me with the flake and wanted to leave this in the best shape I could before departing...